server/db/migrations/20180223115953_isApproved.js

```diff
@@ -0,0 +1,11 @@
+exports.up = (knex, Promise) => {
+  return knex.schema.table('users', (table) => {
+    table.boolean('isApproved')
```

Comment:
Table columns should be snake/train_case. Let's rename to `is_approved` (for the `dropColumn` call below as well). This will likey require code changes elsewhere also.